### PR TITLE
get rid of planning-test-helper in plan-consumer-test

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

File: src/planning/plan/tests/planning-result-test.ts [hunk 1]
```diff
@@ -10,57 +10,71 @@
...
+    const context = await Manifest.load(manifestFilename, loader);
+    const runtime = new Runtime(loader, FakeSlotComposer, context);
+    const arc = runtime.newArc('demo', storageKeyPrefixForTest());
+    const suggestions = await StrategyTestHelper.planForArc(arc);
+
+    assert.isNotEmpty(suggestions);
+    // suggestions.forEach(s => {
+    //   s.relevance = Relevance.create(arc, s.plan);
+    //   s.relevance.apply(new Map([[s.plan.particles[0], [1]]]));
+    // });
```
Comment (mykmartin) [inline src/planning/plan/tests/planning-result-test.ts]: Is this something that needs to be revisited?

File: src/tests/demo-flow-test.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@
 import {assert} from '../platform/chai-web.js';
 import {Loader} from '../platform/loader.js';
 import {Manifest} from '../runtime/manifest.js';
-import {PlanningTestHelper} from '../planning/testing/arcs-planning-testing.js';
+import {PlanningTestHelper} from '../planning/testing/planning-test-helper.js';
```
Comment (mykmartin) [inline src/tests/demo-flow-test.ts]: We're thinking this file can probably just be deleted - the demo flow test hasn't been enabled for a long time (and the other one is a super-trivial load test). We have that wdio tests for some reasonable level of integration coverage.

File: src/planning/plan/tests/planning-result-test.ts (hunk 1 above)
Comment (mmandlis) [reply src/planning/plan/tests/planning-result-test.ts]: removed.

File: src/tests/demo-flow-test.ts (hunk 2 above)
Comment (mmandlis) [reply src/tests/demo-flow-test.ts]: i'll make a separate pr to remove it.

**Pull request merged**